nemo/export/tensorrt_llm.py

```diff
@@ -253,8 +255,8 @@ def export(
                     max_output_len=max_output_len,
                     max_batch_size=max_batch_size,
                     max_prompt_embedding_table_size=max_prompt_embedding_table_size,
-                    tensor_parallel_size=tensor_parallel_size,
-                    pipeline_parallel_size=pipeline_parallel_size,
+                    tensor_parallel_size=tensor_parallelism_size,
+                    pipeline_parallel_size=pipeline_parallelism_size,
```

Comment:
still different naming is used.. (this is minor though).

I think tensor_parallel_size / pipeline_parallel_size would be more common options